Backend/src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs

```diff
@@ -16,6 +16,7 @@ public Startup(IWebHostEnvironment environment)
             Environment = environment;
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "ASP.net needs this to not be static")]
```

Comment:
To be consistent with all the other projects, could this be moved to the top of the file. Taken from the buying Catalogue API

--------------

    [SuppressMessage("Design", "CA1822", Justification = "ASP.Net needs this to not be static")]
    public sealed class Startup

--------------